TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoLinearAPI.kt

```diff
@@ -0,0 +1,91 @@
+package org.firstinspires.ftc.teamcode.api.linear
+
+import com.qualcomm.robotcore.util.Range
+import org.firstinspires.ftc.teamcode.api.TriWheels
+import org.firstinspires.ftc.teamcode.core.API
+import kotlin.math.atan2
+import kotlin.math.sqrt
+
+/**
+ * This is a demo API for driving related to april tags.
+ *
+ * Requires the [TriWheels] API.
+ */
+object DemoLinearAPI : API() {
+    override val isLinear = true
```

Comment:
```suggestion
    // `isLinear` requires that this API is only initialized with a `LinearOpMode`.
    // By enabling it, you can access `this.linearOpMode`.
    override val isLinear = true
```

`isLinear` is the largest thing that differentiates linear APIs from normal ones. I think it deserves a comment explaining what it does.